src/server/services/afis/afis-helpers.ts

```diff
@@ -36,3 +39,43 @@ export async function getAfisApiConfig(
   };
   return getApiConfig('AFIS', additionalConfigWithHeader);
 }
+
+export function getEmandateValidityDateFormatted(dateValidTo: string | null) {
+  const dateValidToFormatted =
+    dateValidTo && !dateValidTo.includes('9999')
+      ? defaultDateFormat(dateValidTo)
+      : dateValidTo
+        ? 'Doorlopend'
+        : null;
+
+  return dateValidToFormatted;
+}
```

Comment:
Ben ik nou gek of is dit hetzelfde?
```suggestion
export function getEmandateValidityDateFormatted(dateValidTo: string | null) {
  return dateValidTo?.includes('9999') ? 'Doorlopend' : dateValidTo;
}
```